src/storage/src/hummock/sstable/multi_builder.rs

```diff
@@ -45,6 +48,8 @@ pub struct CapacitySplitTableBuilder<B> {
     current_builder: Option<SSTableBuilder>,
 
     sstable_store: SstableStoreRef,
+
+    uploading_size: Arc<AtomicUsize>,
```

Comment:
How will `uploading_size` be used?

Reply:
> How will `uploading_size` be used?

to prevent new writes when current `uploading_size` is too big

Reply:
How will we prevent new write? Do we spin loop there waiting for the `uploading_size` to be small again?

Reply:
> How will we prevent new write? Do we spin loop there waiting for the `uploading_size` to be small again?

yes that is what I designed

Reply:
Let's also include the usage `uploading_size` in this PR for reviews. I think it won't make this PR much larger.

Reply:
> Let's also include the usage `uploading_size` in this PR for reviews. I think it won't make this PR much larger.

just pushed a commit to use `uploading_size`